lib/codec.js

```diff
@@ -18,6 +18,7 @@
   exports.encodeBytes = f.lift(doEncodeBytes, _.identity);
   exports.encodeString = f.lift(doEncodeString, _.identity);
   exports.encodeSignedInt = f.lift(doEncodeSignedInt, _.identity);
+  exports.encodeShort = f.lift(doEncodeShort, _.identity);
```

Comment:
Ok, got it. Actually I have added this for being able to add tests for encodeShort as well as for increasing the code coverage for codec.js. But if this will be removed in the future, then I will remove this with corresponding tests.